lib/geminabox/server.rb

```diff
@@ -312,7 +312,7 @@ def remote_gem_list
     end
 
     def combined_gem_list
-      GemListMerge.from(local_gem_list, remote_gem_list)
+      GemListMerge.new(local_gem_list).merge(remote_gem_list, Geminabox.rubygems_proxy_merge_strategy)
```

Comment:
Why isn't the class method being used here? I think this should be:
```ruby
GemListMerge.merge(local_gem_list, remote_gem_list, strategy: Geminabox.rubygems_proxy_merge_strategy)
```

Reply:
End result of a refactoring chain. Will match your style preference.